sekoia_automation/connector.py

```diff
@@ -30,6 +30,8 @@ class DefaultConnectorConfiguration(BaseModel):
 class Connector(Trigger):
     configuration: DefaultConnectorConfiguration
 
+    seconds_without_events = 3600
```

Comment:
You also need to implement the part updating `_last_events`

Reply:
You are right, I saw that `send_records` was calling `send_events` but didn't notice the `push_events_to_intakes` method that didn't.